semconv-incubating/src/main/java/io/opentelemetry/semconv/incubating/HttpIncubatingAttributes.java

```diff
@@ -193,6 +193,13 @@ public final class HttpIncubatingAttributes {
   public static final AttributeKey<String> HTTP_URL = stringKey("http.url");
 
   // Enum definitions
+  /**
+   * Values for {@link #HTTP_REQUEST_METHOD}.
+   *
+   * @deprecated deprecated in favor of stable {@link
+   *     io.opentelemetry.semconv.HttpAttributes.HttpRequestMethodValues} attribute.
+   */
+  @Deprecated
```

Comment:
This is a good example of the deprecated annotation. Javadoc links back to the stable equivalent.